src/XMakeCommandLine/UnitTests/XMake_Tests.cs

```diff
@@ -891,14 +892,15 @@ public void GetCommandLineQuotedExeOnPath()
             {
                 Directory.SetCurrentDirectory(Path.GetDirectoryName(_pathToMSBuildExe));
 
-                output = RunProcessAndGetOutput("msbuild.exe", "\"" + _pathToArbitraryBogusFile + "\"" + " /v:diag", expectSuccess: false);
+                var msbuildParameters = "\"" + _pathToArbitraryBogusFile + "\"" + " /v:diag";
+                output = runMSBuildWithinCoreRun("msbuild.exe", msbuildParameters, false);
             }
             finally
             {
                 Directory.SetCurrentDirectory(current);
             }
 
-            Assert.True(output.IndexOf(_pathToMSBuildExe + " /v:diag " + _pathToArbitraryBogusFile, StringComparison.OrdinalIgnoreCase) >= 0);
+            Assert.True(output.Contains(_pathToMSBuildExe + " /v:diag " + _pathToArbitraryBogusFile));
```

Comment:
You could actually change this to `Assert.Contains`.  If you didn't strip the `OrdinalIgnoreCase` on purpose, there's a bool flag for that.